app/views/invoices/show.html.slim

```diff
@@ -1,37 +1,69 @@
 .page-header
-  .pull-right
+  .btn-group.pull-right
+    = link_to 'Destroy', @invoice, method: :delete,
+      data: { confirm: 'Are you sure?' },
+      class: 'btn btn-danger'
+    = link_to 'Edit', edit_invoice_path(@invoice),
+      class: 'btn btn-default'
+    = link_to 'Download as PDF', invoice_path(@invoice, {format: :pdf}.merge((Rails.env.test?) ? { debug: true } : {})),
```

Comment:
@yunixon I don't think it's a good idea to test PDF with this way. Currently in tests you don't cover case with file downloading, and if there will be some issues with PDF generation - you will never see it in specs.